workflows/charts/tgi/values.yaml

```diff
@@ -18,13 +18,11 @@ nameOverride: ""
 fullnameOverride: ""
 deploy:
   # -- ConfigMap of Environment Variables
-  configMapName: intel-proxy-config
+  configMap:
+    enabled: true
+    name: intel-proxy-config
```

Comment:
Interesting that you reference the intel-proxy-config configMap, but it's not here. I guess that means it's optional?

Reply:
I could probably change it to something more mundane than proxy config